internal/command/postgres/postgres.go

```diff
@@ -191,3 +194,57 @@ func UnregisterMember(ctx context.Context, app *fly.AppCompact, machine *fly.Mac
 
 	return nil
 }
+
+// Runs a command on the specified machine ID in the named app.
+func ExecOnMachine(ctx context.Context, appName, machineId, command string) error {
```

Comment:
It's generally best for functions like this to take the client as an argument.  If this is used during orchestration, we don't need to rebuild the flaps client with each exec.